src/evict/evict_lru.c

```diff
@@ -2529,6 +2529,10 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
             WT_ERR(ret);
         }
 
+        /* FIXME-WT-12905: Prefetch threads are not allowed to be pulled into eviction. */
```

Comment:
Why not return earlier at the start of the function?

Reply:
Excellent point, done in 1d49ae1d15da9090e8242e93ae43a8c496dc51cb.